docs/src/api/class-video.md

```diff
@@ -41,6 +41,18 @@ upon closing the browser context. This method throws when connected remotely.
 Saves the video to a user-specified path. It is safe to call this method while the video
 is still in progress, or after the page has closed. This method waits until the page is closed and the video is fully saved.
 
+## async method: Video.saveAs
+* langs: java
+* since: v1.11
+
+Saves the video to a user-specified path. This must be called after [`method: page.close`], otherwise an error will be thrown. This method waits until the video is fully saved.
+
+## async method: Video.saveAs
+* langs: python
+* since: v1.11
+
+Saves the video to a user-specified path. If using the sync API, This must be called after [`method: page.close`], otherwise an error will be thrown. If using the async API, it is safe to call this method while the video is still in progress, or after the page has closed. This method waits until the page is closed and the video is fully saved.
```

Comment:
```suggestion
Saves the video to a user-specified path. If using the sync API, this must be called after [`method: page.close`], otherwise an error will be thrown. If using the async API, it is safe to call this method while the video is still in progress, or after the page has closed. This method waits until the page is closed and the video is fully saved.
```